deployment/k8s/charts/aggregates/migrationAssistantWithArgo/templates/resources/otel/simpleCollectorAsIndependentDaemonset.yaml

```diff
@@ -24,6 +24,10 @@ spec:
               value: "true"
             - name: OTEL_LOG_LEVEL
               value: "debug"
+            - name: QUALIFIER
+              value: {{ .Values.stageName }}
+            - name: TRACE_SAMPLING_PERCENTAGE
+              value: "1"
```

Comment:
is this 100%, 1%?  I think that we should turn all traces off.  There's a severe performance degradation risk with them being on & customers have never asked about traces, nor have we ever documented them.
This isn't different from our ECS story - but I'd like to eliminate the need to do helm redeployments & I'd rather have people need to redeploy for more debugging info rather than redeploying to fix a performance issue.